include/fastq_conversion.hpp

```diff
@@ -0,0 +1,14 @@
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/io/sequence_file/all.hpp>
+#include <seqan3/io/sequence_file/input.hpp>
+#include <seqan3/io/sequence_file/output.hpp>
+
+void convert_fastq(std::filesystem::path fastq_file, std::filesystem::path out)
```

Comment:
`std::filesystem::path` is defined in header `<filesystem>`. Please include it here in this header file (then it can be omitted in the cpp file).